creator-node/src/fileManager.js

```diff
@@ -86,7 +86,9 @@ async function saveFileToIPFSFromFS (req, srcPath) {
  * @param {String?} fileNameForImage file name if the multihash is image in dir.
  *                  eg original.jpg or 150x150.jpg
  */
-async function saveFileForMultihashToFS (req, multihash, expectedStoragePath, gatewaysToTry, fileNameForImage = null) {
+async function saveFileForMultihashToFS (serviceRegistry, logger, multihash, expectedStoragePath, gatewaysToTry, fileNameForImage = null) {
```

Comment:
nit - lets make this { ipfsLatest } to keep it consistent with the ipfs functions?  also the jsdoc above needs to be updated

Reply:
the reason i can do that in the ipfs functions is bc they don't internally have to pass serviceRegistry anywhere. if i did that here then i wouldn't be able to call `await Utils.ipfsCat(serviceRegistry, logger, multihash, 1000)` which is why i did it this way

good catch on the jsdoc tho, fixed